### ISSUE TITLE
confusion in OhemCrossEntropy loss function

### ISSUE THREAD

**Issue by Srutsth** [Serge-weihao/CCNet-Pure-Pytorch#15]
pred = prob[label, np.arange(len(label), dtype=np.int32)]

What does this line do in OhemCrossEntropy do?

**Comment by Serge-weihao** [comment]
https://github.com/Serge-weihao/CCNet-Pure-Pytorch/blob/ff51a27bc5e111b2e7ddc9258266a2fea4637ac6/loss/loss.py#L68
I guess that means to get the gt class related predicted score for each (valid) pixel.
